bootstrap/bootstrap.py

```diff
@@ -205,6 +205,14 @@ def get_os_release_variable(key):
     )
 
 
+def host_has_systemd() -> bool:
```

Comment:
You've added this function, but it's not used

Reply:
Good eye, whoops. I was contemplating breaking out more pieces to test before I noticed you already have an integration test suite running on GitHub Actions. I've stripped that part out, now to see if the other failures are relevant to this PR or not.